radicle-httpd/src/api/v1/projects.rs

```diff
@@ -303,14 +304,16 @@ async fn readme_handler(
     ];
 
     for path in paths {
-        if let Ok(blob) = radicle_surf::blob::blob(&repo.raw().into(), Some(Sha { sha }), path) {
-            return Ok::<_, Error>(Json(blob));
+        if let Ok(blob) = repo.blob(sha, path) {
+            let response = json!({
+                "path": &path,
+                "readme": blob.content(),
```

Comment:
Again this is to show an example / possibility, not saying the field should be named "readme".

Reply:
Yeah we don't send over binary data, and when it's plain text we convert to utf8 string and send that over.

Reply:
What if a file is big? say 10GB, or 1GB? would we do streaming instead?

Reply:
Hmm, not sure. The UI is not meant to handle files like that really, so I'd say that beyond a certain size we wouldn't display it.

Reply:
Git in general isn't very good with large files, so I don't _think_ we have to try worry about it too much